akka-actor-typed/src/main/scala/akka/actor/typed/internal/ActorContextImpl.scala

```diff
@@ -206,6 +207,14 @@ import akka.util.Timeout
     pipeToSelf((target.ask(createRequest))(responseTimeout, system.scheduler))(mapResponse)
   }
 
+  override def failableAsk[Req, Res](target: RecipientRef[Req], createRequest: ActorRef[StatusResponse[Res]] => Req)(
```

Comment:
I'd prefer something prefixed with `ask`.
`askWithReply` ?

Reply:
`askWithReply` may be confusing because there is a reply when using `ask`

Reply:
I agree `ask` first would be good for discoverability, tricky to keep it short with a suffix though `askThatCanFail`... ;)